src/main/java/com/commercetools/dataimport/DataImportJobConfiguration.java

```diff
@@ -68,37 +71,43 @@ public Flow projectCleanUpFlow(Step ordersDeleteStep, Step cartsDeleteStep, Step
     }
 
     @Bean
-    public Flow projectSetUpFlow(Step projectSettingsStep, Step customerTypeImportStep) {
+    public Flow projectSetUpFlow(Step projectSettingsStep, Step customerTypeImportStep,
+                                 Step taxCategoryImportStep, Step customerGroupImportStep,
+                                 Step channelTypeImportStep, Step productTypeImportStep) {
         final FlagFlowDecider flagFlowDecider = new FlagFlowDecider(projectSetUpFlag);
         return new FlowBuilder<Flow>("projectSetUpFlow")
                 .start(flagFlowDecider).on(FlagFlowDecider.RUN)
                     .to(projectSettingsStep)
                     .next(customerTypeImportStep)
+                    .next(customerGroupImportStep)
+                    .next(taxCategoryImportStep)
+                    .next(channelTypeImportStep)
+                    .next(productTypeImportStep)
                 .from(flagFlowDecider).on(FlagFlowDecider.SKIP)
                     .end()
                 .build();
     }
 
     @Bean
-    public Flow catalogImportFlow(Step productTypeImportStep, Step taxCategoryImportStep,
-                                  Step channelTypeImportStep, Step channelsImportStep, Step customerGroupImportStep,
-                                  Step categoriesImportStep, Step productsImportStep, Step inventoryImportStep) {
+    public Flow catalogImportFlow(Step categoriesImportStep) {
         final FlagFlowDecider flagFlowDecider = new FlagFlowDecider(catalogImportFlag);
         return new FlowBuilder<Flow>("catalogImportFlow")
                 .start(flagFlowDecider).on(FlagFlowDecider.RUN)
-                    .to(customerGroupImportStep)
-                    .next(categoriesImportStep)
-                    .next(taxCategoryImportStep)
-                    .next(channelTypeImportStep)
-                    .next(channelsImportStep)
-                    .next(productTypeImportStep)
-                    .next(productsImportStep)
-                    .next(inventoryImportStep)
+                    //.to(customerGroupImportStep)
+                    .to(categoriesImportStep)
+                    //.next(taxCategoryImportStep)
+                    //.next(channelTypeImportStep)
+                    //.next(channelsImportStep)
+                    //.next(productTypeImportStep)
+                    //.next(productsImportStep)
+                    //.next(inventoryImportStep)
```

Comment:
Instead of disabling all this, we could just put it into another Flow and enable it via configuration. This way the tests below do not need to be disabled.

If you need any help let me know :)